pandas/tests/groupby/test_apply.py

```diff
@@ -866,3 +866,27 @@ def fct(group):
         [[1.0, 2.0], [3.0], [np.nan]], index=pd.Index(["a", "b", "none"], name="A")
     )
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "values",
+    [
+        {
+            "a": [1, 1, 1, 2, 2, 2, 3, 3, 3],
+            "b": [1, pd.NA, 2, 1, pd.NA, 2, 1, pd.NA, 2],
+        },
+        {"a": [1, 1, 2, 2, 3, 3], "b": [1, 2, 1, 2, 1, 2]},
+    ],
+)
+@pytest.mark.parametrize("function", ["mean", "median", "var"])
```

Comment:
Can you use `reduction_func` fixture instead of picking these one-off?

Reply:
I'm having some trouble using this fixture without making the test very complex since the methods each have different output (e.g., sometimes it's empty), some require arguments, etc. Would it be okay if these tests only included the ones affected by the bug?